tests/integration/replication.tcl

```diff
@@ -1659,3 +1665,171 @@ start_server {tags {"repl external:skip"}} {
         }
     }
 }
+start_server {tags {"repl external:skip"}} {
+    set master [srv 0 client]
+    set master_host [srv 0 host]
+    set master_port [srv 0 port]
+
+    start_server {} {
+        set slave [srv 0 client]
+        $slave slaveof $master_host $master_port
+
+    test "Accumulate repl_total_disconnect_time with delayed reconnection" {
+        set unknown_port [get_free_port]
+        wait_for_condition 50 100 {
+            [string match {*master_link_status:up*} [$slave info replication]]
+        } else {
+            fail "Initial replica setup failed"
+        }
+        
+        # Simulate disconnect by pointing to invalid master
+        $slave slaveof $master_host $unknown_port
```

Comment:
use port 0?
```suggestion
        $slave slaveof $master_host 0
```

Reply:
I'll modify your test. It can pass the test on my local machine.
In other tests, we always use `replicaof ip 0` to make the replica connect to an invalid port.